packages/perseus/src/perseus-types.ts

```diff
@@ -909,6 +909,125 @@ export type PerseusGraphTypeRay = {
     startCoords?: CollinearTuple;
 } & PerseusGraphTypeCommon;
 
+type AngleGraphCorrect = {
+    type: "angle";
+    // Whether to show the angle measurements.  default: false
+    allowReflexAngles?: boolean;
+    match: "congruent";
+    // must have 3 coords - ie [Coord, Coord, Coord]
+    coords: [Coord, Coord, Coord];
+};
+
+type CircleGraphCorrect = {
+    type: "circle";
+    center: Coord;
+    radius: number;
+};
+type LinearGraphCorrect = {
+    type: "linear";
+    // expects 2 coords
+    coords: CollinearTuple;
+};
+
+type LinearSystemGraphCorrect = {
+    type: "linear-system";
+    // expects 2 sets of 2 coords
+    coords: CollinearTuple[];
+};
+
+type NoneGraphCorrect = {
+    type: "none";
+};
+
+type PointGraphCorrect = {
+    type: "point";
+    coords: ReadonlyArray<Coord>;
+};
+
+type PolygonGraphCorrect = {
+    type: "polygon";
+    match: "similar" | "congruent" | "approx";
+    coords: ReadonlyArray<Coord>;
+};
+
+type QuadraticGraphCorrect = {
+    type: "quadratic";
+    // expects a list of 3 coords
```

Comment:
I'm not totally convinced we need these comments. :)   The type should, and does, define how many coords we expect.  I think these comments would only be useful if our types don't allow us to be explicit enough (like a case where we _have_ to use `ReadonlyArray<Coord>` but we expect only 2 coords).

Reply:
Sounds good! I can remove them. They were just copied over from where the graph properties were defined.